mocks/mocks.py

```diff
@@ -15,12 +15,39 @@
 
 import ccdproc
 
+import pynbody
+
 import gunagala
 from gunagala.utils import ensure_unit
 
 from mocks.utils import load_yaml_config
 
 
+def read_gadget(data_path):
+    """
+    Reads the raw output of the Genesis simulations
```

Comment:
Use `read_genesis` then, not `read_gadget`. Unless this function works for all gadget, then change this line appropriately.